voter/management/commands/voter_process_snapshot.py

```diff
@@ -145,6 +146,8 @@ def find_existing_instance(ncid):
     Will prefetch all ChangeTracker instances related."""
 
     voter = NCVoter.objects.filter(ncid=ncid).prefetch_related('changelog').first()
+    if voter:
+        assert voter.changelog.filter(op_code=ChangeTracker.OP_CODE_ADD).exists()
```

Comment:
I'm assuming this is a 'should never happen', but I'm slightly worried about it getting triggered, since we have manually deleted many ChangeTracker records. OTOH, I guess we have Sentry installed, so we'll see it now, if it does get triggered. Thoughts?

Reply:
I think its best to leave and see if the data changes you had to make trigger anything. If not, we can remove this after the imports complete. You are right in that it was a "should never happen" that I ran into while code was broken refactoring some of the original import code, but I left it in "just in case" because the failure case was silent.